Documentation/Helm-Charts/operator-chart.md.gotmpl

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
If you [preview the docs](https://rook.io/docs/rook/latest/Contributing/documentation/#local-preview), does the document rendering ignore these .gotmpl files? Or do the .gotmpl files show up in the rendered docs?